packages/react-storage/src/components/StorageBrowser/context/elements/definitions.ts

```diff
@@ -44,6 +44,38 @@ export type IconVariant =
 
 export type IconElementProps = React.ComponentProps<typeof IconElement>;
 
+export const DEFAULT_ICON_PATHS: Record<IconVariant, string> = {
+  'action-queued':
```

Comment:
Think it would be helpful if we add comments above each key mapping to the name of the corresponding material icon